src/Microsoft.AspNet.Testing/project.json

```diff
@@ -7,10 +7,11 @@
     },
     "dependencies": {
         "Microsoft.Dnx.Runtime.Abstractions": "1.0.0-*",
-        "xunit.runner.aspnet": "2.0.0-aspnet-*"
```

Comment:
At one point it wouldn't compile w/o depending on the runner directly. If that's gone away, wunderbar.